packages/expo-camera/android/src/main/java/expo/modules/camera/ExpoCameraView.kt

```diff
@@ -90,6 +92,26 @@ class ExpoCameraView(
     get() = appContext.currentActivity as? AppCompatActivity
       ?: throw Exceptions.MissingActivity()
 
+  val orientationEventListener by lazy {
+    object : OrientationEventListener(currentActivity) {
```

Comment:
Should we recreate the event listener when the activity is recreated by the Android system? Could you test it with the `don't keep activities` option enabled? It may work because the view also should be recreated by the system. However, testing it won't hurt 😅

Reply:
No problem, I'll check it now. Always hate doing anything with the activity 😅